pages/settings/index.vue

```diff
@@ -1,7 +1,204 @@
 <template>
```

Comment:
This file could have been a great pick for a component test. Well, next time :wink: